Rethinking_2/Chp_06.ipynb

```diff
@@ -0,0 +1,3109 @@
+{
```

Comment:
<p>This is neat, thanks for digging that up!</p><p>Could you add datf and causalgraphicalmodels in the environment.yml file of this sub repo please?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/resources/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>